### PR TITLE
6305 update discovery ui

### PR DESCRIPTION
**Pull request by wschaeferB**

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/discovery/DiscoveryEventUtils.java [hunk 1]
```diff
@@ -332,4 +373,33 @@ int getGroupSize() {
...
+
+        /**
+         * Construct a new DetailsVisibleEvent.
+         *
+         * @param isVisible True if the details area should be visible, false
+         *                  otherwise.
+         */
+        DetailsVisibleEvent(boolean isVisible) {
+            showDetailsArea = isVisible;
+            //no arg constructor
```
Comment (APriestman) [inline Core/src/org/sleuthkit/autopsy/discovery/DiscoveryEventUtils.java:392]: Again, it seems like there's an argument given...

File: Core/src/org/sleuthkit/autopsy/discovery/DiscoveryEventUtils.java [hunk 2]
```diff
@@ -70,6 +75,42 @@ FileType getType() {
...
+    static final class PopulateInstancesListEvent {
+
+        private final List<AbstractFile> instances;
+
+        /**
+         * Construct a new PopulateInstancesListEvent.
+         */
+        PopulateInstancesListEvent(List<AbstractFile> files) {
+            instances = files;
+            //no arg constructor
```
Comment (APriestman) [inline Core/src/org/sleuthkit/autopsy/discovery/DiscoveryEventUtils.java:103]: There seems to be an arg though?

File: Core/src/org/sleuthkit/autopsy/discovery/ParentFolderFilterPanel.java [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+            deleteButton.setEnabled(true);
+        }
+    }//GEN-LAST:event_parentListValueChanged
+
+    private void deleteButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_deleteButtonActionPerformed
+        int index = parentList.getSelectedIndex();
+        if (index >= 0) {
+            parentListModel.remove(index);
+        }
+//        validateFields();
```
Comment (APriestman) [inline Core/src/org/sleuthkit/autopsy/discovery/ParentFolderFilterPanel.java:207]: Should probably just delete this (and the one in the next method) if no longer needed

File: Core/src/org/sleuthkit/autopsy/discovery/SizeFilterPanel.java [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+
+/**
+ * Panel to allow configuration of the Size Filter.
+ */
+final class SizeFilterPanel extends AbstractDiscoveryFilterPanel {
+
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * Creates new form SizeFilterPanel=
```
Comment (APriestman) [inline Core/src/org/sleuthkit/autopsy/discovery/SizeFilterPanel.java:37]: Typo (=)

File: Core/src/org/sleuthkit/autopsy/discovery/SizeFilterPanel.java [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+            List<FileSearchData.FileSize> sizes;
+            switch (fileType) {
+                case VIDEO:
+                    sizes = FileSize.getOptionsForVideos();
+                    break;
+                case IMAGE:
+                    sizes = FileSize.getOptionsForImages();
+                    break;
+                case DOCUMENTS:
+                    sizes = FileSize.getOptionsForImages();
```
Comment (APriestman) [inline Core/src/org/sleuthkit/autopsy/discovery/SizeFilterPanel.java:154]: Can we maybe rename the image sizes to be more "default size options" (in various places) so it won't seem weird that we're using it for images and videos?

File: Core/src/org/sleuthkit/autopsy/discovery/DiscoveryEventUtils.java (hunk 2 above)
Comment (wschaeferB) [reply Core/src/org/sleuthkit/autopsy/discovery/DiscoveryEventUtils.java:103]: woopsie

File: Core/src/org/sleuthkit/autopsy/discovery/ParentFolderFilterPanel.java (hunk 3 above)
Comment (wschaeferB) [reply Core/src/org/sleuthkit/autopsy/discovery/ParentFolderFilterPanel.java:207]: agreed, thank you

File: Core/src/org/sleuthkit/autopsy/discovery/SizeFilterPanel.java (hunk 5 above)
Comment (wschaeferB) [reply Core/src/org/sleuthkit/autopsy/discovery/SizeFilterPanel.java:154]: yeah good call

File: Core/src/org/sleuthkit/autopsy/discovery/DiscoveryEventUtils.java (hunk 1 above)
Comment (wschaeferB) [reply Core/src/org/sleuthkit/autopsy/discovery/DiscoveryEventUtils.java:392]: yeah woops

**Pull request merged**